core/controllers/editor.py

```diff
@@ -334,7 +332,7 @@ def get(self, exploration_id):
                 exploration_id, apply_draft=apply_draft, version=version))
```

Comment:
No. As it has to work for guest users as well. (It's a part of playing exploration)